components/messaging/ConversationCard.tsx

```diff
@@ -49,27 +53,26 @@ const ConversationCard: React.FC<ConversationCardProps> = ({
       data-testid='card-wrapper'
       onClick={clickHandler}
     >
-      <div className='relative h-[65px] w-[65px]'>
+      <div className='relative h-[65px] w-[65px] flex-shrink-0'>
         {notificationList.some(
           (conversation) => conversation === conversationId
-        ) && <div className='notification-dot'></div>}
-        {partnerAvatar ? (
-          <Image
-            src={partnerAvatar}
-            fill
-            className='flex-shrink-0 rounded-full'
-            alt={partnerUsername}
-            sizes='(max-width: 640px) 50px, (max-width: 768px) 75px, 100px'
-          />
-        ) : (
-          <Image
-            src={defaultProfileImage}
-            fill
-            className='flex-shrink-0 rounded-full'
-            alt={partnerUsername}
-            sizes='(max-width: 640px) 50px, (max-width: 768px) 75px, 100px'
-          />
+
+        ) && (
+          <div
+            className='relative left-12 z-50 h-4 w-4 rounded-full 
+              border-2 border-green-700 bg-[#54BB89] shadow-lg outline-4 outline-black'
+          ></div>
         )}
```

Comment:
Thanks Anna, just one thing on these lines - it looks like we're losing the 'notification-dot' class here to instead declare the same styling with tailwind inline. We should stick to calling the globally defined class because this is used in several components and needs to be easily edited in future, so I'm making here an inline commit suggestion to revert this change

```suggestion
        ) && <div className='notification-dot' /> )}
```

Reply:
sorry, I tried to commit the suggestion and it didn't go through. I'll try one more time